airflow/providers/google/cloud/transfers/sftp_to_gcs.py

```diff
@@ -130,18 +160,24 @@ def execute(self, context: Context):
 
             for file in files:
                 destination_path = file.replace(base_path, self.destination_path, 1)
-                self._copy_single_object(gcs_hook, sftp_hook, file, destination_path)
+                if self.use_stream:
+                    self._stream_single_object(sftp_hook, gcs_hook, file, destination_path)
+                else:
+                    self._copy_single_object(sftp_hook, gcs_hook, file, destination_path)
```

Comment:
```suggestion
                transfer_single_object = self._stream_single_object if self.use_stream else self.self._copy_single_object
                transfer_single_object(sftp_hook, gcs_hook, file, destination_path)
```

not sure whether it's better, but I might try something like this to reduce the mental work of comparing the arguments